src/main/java/com/seailz/discordjar/model/channel/internal/GuildChannelImpl.java

```diff
@@ -55,4 +68,149 @@ public boolean nsfw() {
     public DiscordJar discordJv() {
         return this.discordJar;
     }
+
+    /**
+     * Creates an Incoming Webhook for this channel.
+     *
+     * @param name The name of the Webhook.
+     * @return The created Webhook.
+     */
+    @Override
+    public IncomingWebhook createWebhook(String name) {
+        return createWebhook(name, null);
+    }
+
+    /**
+     * Creates an Incoming Webhook for this channel.
+     *
+     * @param name The name of the Webhook.
+     * @param avatar The default avatar for the Webhook.
+     * @return The created Webhook.
+     */
+    @Override
+    public IncomingWebhook createWebhook(String name, File avatar) {
+        try {
+            ImageUtils.Image avatarImage = null;
+            if (avatar != null) {
+                BufferedReader reader = new BufferedReader(new FileReader(avatar));
+
+                StringBuilder imageData = new StringBuilder();
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    imageData.append(line);
+                }
+
+                String encodedData = Base64.encodeBase64String(imageData.toString().getBytes());
+
+                String[] splitFileName = avatar.getName().split("\\.");
+                String fileFormat = splitFileName[splitFileName.length - 1];
+
+                avatarImage = ImageUtils.createImageData(ImageUtils.ImageFormat.of(fileFormat), encodedData);
```

Comment:
You're encoding twice here. Raw data should be passed to the createImageData method instead of base64 data.
```suggestion
                avatarImage = ImageUtils.createImageData(ImageUtils.ImageFormat.of(fileFormat), imageData.toString().getBytes());
```